omf/fileio/geoh5.py

```diff
@@ -647,18 +647,23 @@ def collect_omf_attributes(element: MappedData, **kwargs) -> dict:
         if not element.legends:
             return kwargs
 
-        value_map = {
-            count + 1: str(val) for count, val in enumerate(element.legends[0].values)
-        }
-        color_map = np.vstack(
-            [
-                np.r_[count + 1, val, 1.0]
-                for count, val in enumerate(element.legends[1].values)
-            ]
-        )
+        value_map = {}
+        color_map = [np.r_[0, 0, 0, 0, 0]]
```

Comment:
Question here:

what if unknown somewhere else than index=0.

Could happened if GA change a bit. 

Could be nice to map colormap ased on the index which are always sorted.

Reply:
The "unknown" is dealt with after if it exists on the omf file, but I see your point. The dict for value_map could be instantiated the same.